release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -32,6 +32,16 @@ Because only {op-system-base} 7.9 or later is supported for compute machines, yo
 
 With the release of {product-title} 4.8, version 4.5 is now end of life. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShift Container Platform Life Cycle Policy].
 
+[id="ocp-4-8-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties.
+
+As part of that effort, with this release the following changes are in place:
+
+* Our OpenShift Docs GitHub repository 'master' branch has been renamed to 'main'.
```

Comment:
This doesn't appear to be one of the places that the style guide allows use of "our." I'd also consider adding a link to the repo.

```suggestion
* The OpenShift Docs GitHub repository 'master' branch has been renamed to 'main'.
```